streaming/python/tests/test_word_count.py

```diff
@@ -45,14 +46,23 @@ def sink_func(x):
                 (old_value[0], old_value[1] + new_value[1])) \
         .sink(sink_func)
     ctx.submit("word_count")
-    import time
-    time.sleep(3)
+
+    def check_succeed():
+        if os.path.exists(sink_file):
+            import time
+            time.sleep(3)  # Wait all data be written
+            with open(sink_file, "r") as f:
+                result = f.read()
+                assert "a:2" in result
```

Comment:
If the assertion fails, the exception will be propagated and the test case will still fail.

Reply:
If the assertion fails, it indicates some srrors occored, the test should fail in that case

Reply:
What if the file is just created and the content is to be written?

Reply:
Tha't why we wait 3 seconds